src/django/contricleaner/lib/parsers/file_parser.py

```diff
@@ -10,7 +9,7 @@
 
 class FileParser(ABC):
     '''
-    Abstract FileParser class for parsers related to file parsing.
+    Interface FileParser for parsers related to file parsing.
```

Comment:
I realized that it is still an abstract class since it has implemented the `__init__` method. Please, leave the description as it was.